### PR TITLE
Use rosie to generate factories in apps/

### PR DESCRIPTION
**Pull request by maddiedierker**
This PR introduces the [rosie](https://github.com/rosiejs/rosie) package (a lib for unit test factories in js) and uses it in an example test file.

### CODE REVIEW

File: apps/test/unit/templates/manageStudents/manageStudentsReduxTest.js [hunk 1]
```diff
@@ -30,91 +31,10 @@ import manageStudents, {
...
-    loginType: 'picture',
-    secretWords: 'wizard',
-    secretPictureName: 'wizard',
-    secretPicturePath: '/wizard.jpg',
-    sectionId: 53
-  }
+const sectionLoginData = {
+  1: sectionLoginFactory.build({id: 1, name: 'StudentNameA', sectionId: 53}),
+  2: sectionLoginFactory.build({id: 2, name: 'StudentNameC', sectionId: 53}),
+  3: sectionLoginFactory.build({id: 3, name: 'StudentNameD', sectionId: 53})
```
Comment (maddiedierker) [inline apps/test/unit/templates/manageStudents/manageStudentsReduxTest.js:37]: note that even though `sectionId` is not defined as an `attr` in the `sectionLoginFactory`, we can still include it when we build an instance of our factory object.

**Comment by maddiedierker** [comment]
@joshlory @islemaster PTAL!

**Review by islemaster — APPROVED** [review]
:+1: :tada:

**Comment by davidsbailey** [comment]
This looks awesome!

**Pull request merged**